packages/api-v2/src/hooks/useRevokeConnectedApps.ts

```diff
@@ -0,0 +1,22 @@
+import useAuthorize from './useAuthorize';
+import useMutation from '../useMutation';
+
+/** A custom hook for revoking access of particular app. */
+const useRevokeConnectedApps = () => {
+    const { isSuccess } = useAuthorize();
+    const { mutate, ...rest } = useMutation('revoke_oauth_app');
+    const revokeMutate = (appId: number) => {
+        if (isSuccess) {
```

Comment:
Is it necessary to check isSuccess here, This api call will happen on user event right ?
AuthProvider would automatically handle the authorization

Reply:
By right, yes the user will be able to call for revoke access only if the user  is in app hence, is authorized. Not sure, why we have authorized WS call for this in production. I assumed we add this authorized check because this was a call that required authorization. If not necessary i can remove this 🙏